### PR TITLE
LPD-42569 Create playwright asset-categories-admin-web project

### PR DESCRIPTION
**Pull request by DiegoHu97**
LPD-42569 Create playwright asset-categories-admin-web project
# What is this trying to solve?
Create the playwrigt module for Asset Categories 

[Link to ticket](https://liferay.atlassian.net/browse/LPD-42569)

# How am I fixing it?

- I have created the module and a first test that includes the happy path for vocabularies (add, edit and delete), creating it bu the UI. This is just the base to test that the Ui works fine. 
- For further tests, for example deleting multiple vocabularies, we should use the API for the vocabularies creation that is faster. For asset categories it should be the same, we create the vocabulary by API and then we continue creating the category.

# How can I test it?
Run the test 
image

### CODE REVIEW

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:content-management-playwright

**Comment by boton** [comment]
ci:stop

**Review by boton — CHANGES_REQUESTED** [review]
Nice work here @DiegoHu97 

I left some comments and I realized that we are not adding the project to the CI

https://liferay.atlassian.net/wiki/spaces/ENGLIMA/pages/3365044312/WIP+How+to+Add+a+New+Playwright+Module
image

**Comment by DiegoHu97** [comment]
Hey @boton Thanks for the review, i've just updated the PR 
image

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

File: modules/test/playwright/playwright.config.ts [hunk 1]
```diff
@@ -176,6 +177,7 @@ export default defineConfig({
 		headlessBuilderWebConfig,
 		iframeWebConfig,
 		itemSelectorTaglibConfig,
+		assetCategoriesAdminWebConfig,
```
Comment (boton) [inline modules/test/playwright/playwright.config.ts:180]: Sort this, move `assetCategoriesAdminWebConfig` between

```js
		assetPublisherWebConfig,
		assetTagsAdminWebConfig,
```

**Review by boton — APPROVED** [review]
LGTM!

Only I left a small change can you please sort the configs projects array

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**